### PR TITLE
Validation week

### PR DESCRIPTION
**Pull request by lopezt-unity**
### Purpose of this PR

Various fixes from the Validation week on PB6.

More info in that document:
https://docs.google.com/document/d/1uWb6afHSvFSXG7yuzkP7deBRgHGeIqvMwL-6B7YSkD4/edit

### Links

**Jira:**

### Comments to Reviewers

[List known issues, planned work, provide any extra context for your code.]

### CODE REVIEW

File: Editor/MenuActions/Object/MirrorObjects.cs [hunk 1]
```diff
@@ -157,40 +158,40 @@ protected override ActionResult PerformActionImplementation()
          */
         public static ProBuilderMesh Mirror(ProBuilderMesh pb, Vector3 scale, bool duplicate = true)
         {
-            ProBuilderMesh mirredObject;
```
Comment (karl-) [inline Editor/MenuActions/Object/MirrorObjects.cs:160]: are we sure this isn't a mired object :stuck_out_tongue_closed_eyes:

File: Editor/Overlays/ActionSettings.cs [hunk 2]
```diff
@@ -105,23 +127,39 @@ void UpdatePreview()
...
+            s_CanTriggerNewAction = false;
             if (s_ActionSettingsOverlay != null)
                 s_ActionSettingsOverlay.Validate();
 
             // Creating the overlay based on the action to fill the settings
             s_ActionSettingsOverlay = new MenuActionSettingsOverlay(action, hasPreview);
+
+            // Ensure we are not calling the action again in the same frame
+            // Needed because Menu Items are called once per selected object while PB actions are acting on the entire selection at once
+            EditorApplication.delayCall += () => { s_CanTriggerNewAction = true; };
```
Comment (karl-) [inline Editor/Overlays/ActionSettings.cs:161]: thank you for adding a comment explaining why `delayCall` is necessary!

**Review by karl- — APPROVED** [review]
Did a proof-reading look through, but did not pull or test manually any changes.

**Review by antoine-unity — APPROVED** [review]

**Review by modrimkus-unity — APPROVED** [review]
Looked through the code and pulled the changes for some sanity testing, especially regarding the more serious bugs - looks good.

**Pull request merged**